pkg/drivers/kic/oci/oci.go

```diff
@@ -164,6 +165,23 @@ func CreateContainerNode(p CreateParams) error {
 	if err != nil {
 		return errors.Wrap(err, "create a kic node")
 	}
+
+	checkRunning := func() error {
+		s, err := ContainerStatus(p.OCIBinary, p.Name)
+		if err != nil {
+			return fmt.Errorf("temporary error checking status for %q : %v", p.Name, err)
+		}
+		if s != "running" {
```

Comment:
nit: wdyt of having all possible states as constants? That way they can be reused later on.

Reply:
Thats a good idea ! eventually we should have a container status with libmachine's state strcut I will create an issue for this and do it in a following up PR